airbyte-integrations/bases/source-acceptance-test/tools/strictness_level_migration/README.md

```diff
@@ -0,0 +1,36 @@
+# Tooling for `test_strictness_level` migration
+
+This directory contains scripts that can help us manage the migration of connectors's `acceptance-test-config.yml` to `high` test strictness level.
+Before running these scripts you need to set up a local virtual environment in the **current directory**:
+```bash
+python -m venv .venv
+source .venv/bin/activate
+pip install -r requirements.txt
+```
+## Requirements
+* [GitHub CLI](https://cli.github.com/) (`brew install gh`)
+
+## Create migration issue for GA connectors (`create_issues.py`)
+This script will create one issue per GA connectors to migrate to `high` test strictness level.
+
+### What it does:
+1. Find all GA connectors in `../../../../../airbyte-config/init/src/main/resources/seed/source_definitions.yaml`
+2. Generate an issue content (title, body, labels, project), using `./templates/issue.md.j2`
+3. Find and already existing issue with the same title.
+4. Create the issue and return its url if it does not exist.
+
+Issues get created with the following labels:
+* `area/connectors`
+* `team/connectors-python` 
+* `type/enhancement` 
+* `test-strictness-level`
+
+Issues are added to the following project: `SAT-high-test-strictness-level`
```

Comment:
Nice- I'm guessing this was easier to do than adding to a Zenhub epic?

Reply:
Yes :) I'll add to a Zenhub epic manually afterward, but it's a good way of quickly finding everything created by this script.